google-cloud-storage/src/test/java/com/google/cloud/storage/BlobInfoTest.java

```diff
@@ -66,6 +66,8 @@
   private static final String KEY_SHA256 = "keySha";
   private static final CustomerEncryption CUSTOMER_ENCRYPTION =
       new CustomerEncryption(ENCRYPTION_ALGORITHM, KEY_SHA256);
+  private static final String STORAGE_CLASS = "COLDLINE";
```

Comment:
We should make storage class string constants available (as per https://github.com/GoogleCloudPlatform/google-cloud-dotnet/pull/766/files#diff-e26b094819cdfbac4ddb7249471814abR23)